src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.PatternLocalRewriter.cs

```diff
@@ -472,7 +470,10 @@ protected BoundExpression MakeRelationalTest(SyntaxNode syntax, BoundExpression
                 Debug.Assert(savedInputExpression != null);
                 return decisionDag;
 
-                bool usesOriginalInput(BoundDecisionDagNode node)
+                static bool isWhenClause(BoundDecisionDagNode node) =>
```

Comment:
>isWhenClause [](start = 28, length = 12)

Consider reverting changes in this file, or renaming this function to reflect significance of the ConstantValue.